RestApi/Sepes.RestApi/Services/AzureService.cs

```diff
@@ -146,6 +150,49 @@ await _azure.NetworkSecurityGroups
                 .Attach()
                 .ApplyAsync();
         }
+        public async Task NsgApplyBaseRules(string securityGroupName, string resourceGroupName)
+        {
+            await _azure.NetworkSecurityGroups
+            .GetByResourceGroup(resourceGroupName, securityGroupName)
+            .Update()
+            .DefineRule("DenyInbound")
+            .DenyInbound()
+            .FromAnyAddress()
+            .FromAnyPort()
+            .ToAnyAddress()
+            .ToAnyPort()
+            .WithAnyProtocol()
+            .WithPriority(4050)
+            .Attach()
+            .DefineRule("AllowVnetInBound2")
+            .AllowInbound()
+            .FromAddress("VirtualNetwork")
+            .FromAnyPort()
+            .ToAddress("VirtualNetwork")
+            .ToAnyPort()
+            .WithAnyProtocol()
+            .WithPriority(4000)
+            .Attach()
+            .DefineRule("DenyOutbound")
+            .DenyOutbound()
+            .FromAnyAddress()
+            .FromAnyPort()
+            .ToAnyAddress()
+            .ToAnyPort()
+            .WithAnyProtocol()
+            .WithPriority(4050)
+            .Attach()
+            .DefineRule("AllowVnetoutBound2")
+            .AllowOutbound()
+            .FromAddress("VirtualNetwork")
+            .FromAnyPort()
+            .ToAddress("VirtualNetwork")
+            .ToAnyPort()
+            .WithAnyProtocol()
+            .WithPriority(4000)
+            .Attach()
+            .ApplyAsync();
```

Comment:
This have be that builder pattern taken to its logical extreme. Should be split up into something smaller.